transhumandesign/kag-base
# Keg couldn't hit spikes

## Status

(pick one)

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Someone forgot to add break; so keg couldn't hit spikes.

## Steps to Test or Reproduce

1. Place spikes
2. Explode keg
3. Check that spikes are destroyed
